src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2689,6 +2695,13 @@ namespace PurchasedAirManager {
                 if (Contaminant.GenericContamSimulation) {
                     Node(InNodeNum).GenContam = Node(ZoneNodeNum).GenContam;
                 }
+                if (PurchAir(PurchAirNum).EMSOverrideMdotOn) {
+                    PurchAir(PurchAirNum).EMSValueMassFlowRate = 0.0;
```

Comment:
this looks odd, EMS programs would be setting the value here.

Reply:
@EnergyArchmage What do you suggest? My intent was to set the value=0, since no cooling and heating.

Reply:
The intent with EMS is let the user's program decide what to do.  This will override whatever the user may be trying to do. The flow should be set to whatever value the user has set in EMSValueMassFlowRate regardless of zone loads.

Reply:
@EnergyArchmage Are you OK with the following code with zero SupplyMassFlowRate?

                if (PurchAir(PurchAirNum).EMSOverrideMdotOn) {
                    SupplyMassFlowRate = PurchAir(PurchAirNum).EMSValueMassFlowRate;
                }

Reply:
@EnergyArchmage A related question is: If the unit is scheduled off, should the EMSOverride be accepted or ignored?